resources/js/src/app/helper/HeaderScroller.js

```diff
@@ -54,21 +54,44 @@ export default class HeaderScroller
      */
     initialize()
     {
-        this.collectHeaderElementHeights();
-        this.updateZIndexes();
 
-        // Initialize only, if the user has scrolled down from the top and is not in the shopbuilder.
-        if (!App.isShopBuilder && window.pageYOffset > 0)
+
+        this.addBrowserClass();
+        const headerNames = ["default-header", "sticky-top"];
```

Comment:
**question**: Isn't it enough to check for `sticky-top` here? Both the ShopBuilder header and the default header will use the `sticky-top` class, after all.